doc/api/url.md

```diff
@@ -235,6 +235,19 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. 
+Some protocols like `https`, `file`, `ftp`, `wss` and `gopher` have special cases.
+
+For example:
+
+```js
+url.resolve('https://foo.tld', 'bar')   // 'https://foo.tld/bar'
+url.resolve('wss://foo.tld', 'bar')      // 'wss://bar'
```

Comment:
@lpinca Please keep mentioning even the tiniest of mistakes I do.  I am a beginner,I am learning a lot in this process. 
I am going to fix all that you said.